### PR TITLE
Adding GetAccount request utils and ParsableEntity

### PR DESCRIPTION
**Pull request by terranmoccasin**
- Porting over generic `ParsableEntity` from `whirlpools-sdk` - https://github.com/orca-so/whirlpools/blob/main/sdk/src/network/public/parsing.ts#L19
- Adding utility methods for making bulk requests for getting Solana `AccountInfo` objects and parsing them

### CODE REVIEW

File: packages/common-sdk/src/web3/network/parsing.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    try {
+      const buffer = MintLayout.decode(data);
+      const mintInfo: MintInfo = {
+        mintAuthority:
+          buffer.mintAuthorityOption === 0 ? null : new PublicKey(buffer.mintAuthority),
+        supply: u64.fromBuffer(buffer.supply),
+        decimals: buffer.decimals,
+        isInitialized: buffer.isInitialized !== 0,
+        freezeAuthority:
+          buffer.freezeAuthority === 0 ? null : new PublicKey(buffer.freezeAuthority),
```
Comment (yugure-orca) [inline packages/common-sdk/src/web3/network/parsing.ts:61]: ```freezeAuthority === 0``` should be ```freezeAuthorityOption === 0```
(comming PR for whirlpools repo will contain this fix)

**Review by yugure-orca — COMMENTED** [review]
Helpful porting! 🙂

I think creating some small test codes for newly or modified codes will help us in the future.
As the importance of common-sdk increases, the importance of regression testing will also increase.

File: packages/common-sdk/src/web3/network/parsing.ts (hunk 1 above)
Comment (terranmoccasin) [reply packages/common-sdk/src/web3/network/parsing.ts:64]: Sorry not sure I completely follow. Should I wait for a PR in the Whirlpools repo?

**Comment by terranmoccasin** [comment]
Thanks for the review @yugure-orca! I added tests for the new utils - ready for another look :)

**Review by yugure-orca — APPROVED** [review]
Thanks for the adding the test cases.
As mentioned above, we don't need to wait the update of whirlpools repo, and we can fix the bug in this repo now.

**Pull request merged**